test/reduced.js

```diff
@@ -0,0 +1,27 @@
+var assert = require('assert');
+
+var R = require('..');
+
+
+describe('reduced', function() {
+  it('wraps a value', function() {
+    // white box test.
+    var v = {};
+    assert.strictEqual(R.reduced(v)['@@transducer/value'], v);
+  });
+
+  it('flags value as reduced', function() {
+    // white box test.
+    assert.strictEqual(R.reduced({})['@@transducer/reduced'], true);
+  });
+
+  it('shortcircuits reduce', function() {
+    // black box test.
+    assert.strictEqual(
+      R.reduce(
+        R.pipe(R.add, R.ifElse(R.lte(10), R.reduced, R.identity)),
```

Comment:
for a unit test i think we're better off writing out a lambda here, rather than having internal dependencies